tsconfig.json

```
@@ -18,4 +18,4 @@
     "target": "es2021"
   },
   "include": ["src"]
-}
+}
```

let's revert this change.

```suggestion
}

```